cmd/build/v2/build.go

```diff
@@ -191,7 +191,7 @@ func (bc *OktetoBuilder) Build(ctx context.Context, options *types.BuildOptions)
 			}
 
 			if !okteto.Context().IsOkteto && buildSvcInfo.Image == "" {
-				return fmt.Errorf("'build.%s.image' is required if your cluster doesn't have Okteto installed", svcToBuild)
+				return fmt.Errorf("'build.%s.image' is required if your context doesn't have Okteto installed", svcToBuild)
```

Comment:
I am not very sure about this change. The current `` message is the one used at various points when trying to perform a certain type of operation on a cluster that does not have okteto installed. If we change this here we should change it at the other points, for consistency. @jLopezbarb

Reply:
This was a decission that came from the marketing team given that there were issues understanding what is a cluster vs a context. I agree that we should change all other messages from cluster to context. @minhoryang Are you ok with doing those changes?

Reply:
@jLopezbarb @AdrianPedriza Yes, it's okay to replace all `cluster`-mentioning messages. but need to grab these sentences first. I'll try to dive in.

Reply:
Changing `cluster` to `context`:
- `pkg/errors/errors.go`: (I think) not okay to change.
  - [ ] ErrNotOktetoCluster
  - [ ] ErrDivertNotSupported
  - [ ] ErrContextIsNotOktetoCluster
- `cmd/kubeconfig.go`: (I think) not okay to change.
- `cmd/build/v2/build.go`: okay to change.
- `cmd/context/context.go`: [partially](https://github.com/okteto/okteto/blob/master/cmd/context/context.go#L67) okay to change. it explains [the difference between `context` and `cluster`](https://github.com/okteto/okteto/blob/master/cmd/context/context.go#L27-L40).
- `cmd/context/create.go`: (I think) not okay to change. it explains [the difference between `context` and `cluster`](https://github.com/okteto/okteto/blob/master/cmd/context/create.go#L89-L110)
- `cmd/context/update-kubeconfig.go`: (I think) not okay to change.
- ...

Changing `okteto cluster` to `okteto context` might be good.

Reply:
I made a simple commit for reviewing: https://github.com/okteto/okteto/pull/4011/commits/08b03021773f4e1ec79fb6bc12884e5e93bde39b